codecept.conf.ts

```diff
@@ -0,0 +1,55 @@
+exports.config = {
+  output: 'reports',
+  helpers: {
+    Playwright: {
+      browser: 'chromium',
+      url: 'https://www.saucedemo.com',
+      show: true,
```

Comment:
Вот тут нужно сделать 
```suggestion
      show: process.env !== 'CI',
```

Так локально браузер будет показан, а в CI не будет пытаться запуститься.

Думаю это должно помочь испрашить ошибку `codeceptjs` в `CI`

Часть ошибки:
```
╔════════════════════════════════════════════════════════════════════════════════════════════════╗
║ Looks like you launched a headed browser without having a XServer running.                     ║
║ Set either 'headless: true' or use 'xvfb-run <your-playwright-app>' before running Playwright. ║
║                                                                                                ║
║ <3 Playwright Team                                                                             ║
╚════════════════════════════════════════════════════════════════════════════════════════════════╝
```